Symfony/CS/Tests/RulesSetTest.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+use Symfony\CS\FixerFactory;
+use Symfony\CS\RulesSet;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+final class RulesSetTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCreate()
+    {
+        $rulesSet = RulesSet::create();
+
+        $this->assertInstanceOf('Symfony\CS\RulesSet', $rulesSet);
+    }
+
+    /**
+     * @dataProvider provideAllRulesFromSets
+     */
+    public function testIfAllRulesInSetsExists($rule)
+    {
+        $factory = new FixerFactory();
+        $factory->registerBuiltInFixers();
+
+        $fixers = array();
+
+        foreach ($factory->getFixers() as $fixer) {
+            $fixers[$fixer->getName()] = $fixer;
+        }
+
+        $this->assertArrayHasKey($rule, $fixers);
+    }
+
+    public function provideAllRulesFromSets()
+    {
+        $cases = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
+            $cases = array_merge($cases, RulesSet::create(array($setName => true))->getRules());
+        }
+
+        return array_map(function ($item) { return array($item); }, array_keys($cases));
+    }
+
+    /**
+     * @expectedException        \UnexpectedValueException
+     * @expectedExceptionMessage Set "@foo" does not exist.
```

Comment:
Sure. I don't like the idea of silently ignoring, but perhaps exceptions are too harsh. We really need some kind of warning system.